modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-flowcollector-about{context}"]
+
+= About the FlowCollector resource 
+The  `FlowCollector` resource is deployed at the time of installation. You can edit yaml directly in the {product-title} web console. In the web console, navigate to *Operators* -> *Installed Operators*. Then under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. Next, select *cluster* then select the *YAML* tab. There, you can modify the `FlowCollector` resource to configure the Network Observability operator. Some of the notable configurations are documented in this documentation. 
+
+The following example shows a sample `FlowCollector` resource for {product-title} Network Observability operator:
+[id="network-observability-flowcollector-configuring-about-sample_{context}"]
+.Sample `FlowCollector` resource
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF                                <1>
+    ebpf:
+      sampling: 50                            <2>
+      logLevel: info
+      privileged: false
+      resources:
+        requests:
+          memory: 50Mi
+          cpu: 100m
+        limits:
+          memory: 800Mi
+      kafkaBatchSize: 10485760
```

Comment:
Can you remove `kafkaBatchSize`? It's not useful as this sample config doesn't use Kafka, so it could be confusing for user to see it mentioned there.

```suggestion
```

Reply:
Addressed